docker-publish.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [ "A$1" != "Alocal" ]; then
```

Comment:
Genuinely curious, why not `"$1" != "local"`?

Reply:
my intension was to run `./docker-publish.sh` to publish to Docker Hub, and `./docker-publish.sh local`, to just build it locally, so when there is no argument in your case it would throw an error(but it would still continue), but this way I'm avoiding additional `if` and it does not throw anything

Reply:
Apols but I'm a little lost so i ran a test. I didn't get an error in my case.

```
#!/bin/bash


if [ "$1" != "local" ]; then
	echo "Logging into docker"
fi

echo "Building stuff"
```
`./test.sh` outputs:
```
Logging into docker
Building stuff
```
`./test.sh local` outputs:
```
Building stuff
```

Reply:
oh, my mistake I was testing at first with `if [ $1 != "local" ]; then` and that's why I've got this error:
`./docker-publish.sh: line 3: [: !=: unary operator expected`, thank you for pointing this out!